src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: inline;
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  &__star {
```

Comment:
seems like we shouldn't have removed this background-repeat property)
let's bring it back)
![image](https://github.com/mate-academy/layout_product-cards/assets/108538921/f0d795c3-1c81-4266-a940-44a6c1960f5b)